app/views/groupTab/components/MemberList.js

```diff
@@ -1,25 +1,44 @@
-'use strict'
-import React from 'react'
-import { Text, Button, Container, Content, List, ListItem } from 'native-base'
-import { connect } from 'react-redux'
+'use strict';
+import React from 'react';
+import { Text, Button, Container, Content, List, ListItem } from 'native-base';
+import { connect } from 'react-redux';
+import { fetchGroupUsers } from '../../../redux/reducers/users';
+
+class UserList extends React.Component {
+  componentDidMount() {
+    this.props.fetchGroupUsers();
+  }
 
-export default class TaskList extends React.Component {
   render() {
+    const { allUsers } = this.props;
     return (
       <Container>
         <Content>
           <List>
-            <ListItem>
-              <Text>MEMBER NAME</Text>
-              <Text>IMAGE</Text>
-            </ListItem>
-            <ListItem>
-              <Text>MEMBER NAME</Text>
-              <Text>IMAGE</Text>
-            </ListItem>
+            {allUsers.length ?
+              allUsers.map(user => {
+                return (
+                  <ListItem key={user.id}>
+                    <Text>{user.name}</Text>
+                  </ListItem>
+                );
+              }) : null}
           </List>
         </Content>
       </Container>
-    )
+    );
   }
 }
+
+export default connect(
+  state => {
+    return {
+      allUsers: state.users.groupUsers
```

Comment:
this looks good.  as long as allUsers is set correctly in your reducer initial state you'll be fine.